web-ui/src/pages/IntroductionSurveyPage.css

```diff
@@ -1,3 +1,45 @@
+.aboutYourself {
+  font-size: 25px;
+}
+
+.aboutYouInput {
+  margin-top: 2em;
+  position: absolute;
```

Comment:
Absolute positioning should be avoided in general. We can come back to reform this css when we get to this page.